browse/domain/category.py

```diff
@@ -0,0 +1,81 @@
+"""Class that represents a single category."""
+
+from typing import Union, List
+from dataclasses import dataclass, field
+
+from arxiv import taxonomy
```

Comment:
I agree that much of this should go in arxiv-base. The ids and categories used by arXiv do not operate in obvious ways and it would be an unnecessary expense to duplicate them. 

Security through API keys is a better way to prevent the browse system from performing unauthorized actions on the file system. If we are confident that browse cannot access the file management system we don't need to care if it has the module on it's load path or not.